### PR TITLE
create add book layout

### PR DESCRIPTION
**Pull request by ghost**
## Description
A few sentences describing the overall goals of the pull request's commits.
## Reference Requirements
- Product Backlog ID: #PBI-3
- Task ID: #3.1

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

File: resources/views/backend/books/create.blade.php [hunk 1]
```diff
@@ -0,0 +1,101 @@
+@extends('backend.layouts.main')
+@section('title', __('Add book'))
+@section('content')
+<!-- Content Wrapper. Contains page content -->
+<div class="content-wrapper">
+    <!-- Content Header (Page header) -->
+  <section class="content-header">
+    <h1>
+      Book
```
Comment (at-annguyen) [inline resources/views/backend/books/create.blade.php]: Localization h1

File: resources/views/backend/books/create.blade.php [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+  <section class="content">
+    <div class="row">
+      <div class="col-md-8 col-md-offset-2">
+        <div class="box box-info">
+          <!-- /.box-header -->
+          <div class="box-body pad">
+            <form action="" method="POST" enctype="multipart/form-data">
+              {{csrf_field()}}
+              <div class="form-group">
+                <label for="name">Name</label>
```
Comment (at-annguyen) [inline resources/views/backend/books/create.blade.php]: Localization

File: resources/views/backend/books/create.blade.php [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+                <input name="name" type="text" class="form-control" id="name" placeholder="{{ __('Enter name') }}" value="{{ old('name') }}">
+                @if($errors->first('name'))
+                  <span class="text-danger">{{ $errors->first('name') }}</span>
+                @endif
+              </div>
+              <div class="form-group">
+                <label for="name">{{ __('Category') }}</label>
```
Comment (at-annguyen) [inline resources/views/backend/books/create.blade.php]: for="category"

File: resources/views/backend/books/create.blade.php [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+              <div class="form-group">
+                <label for="author">{{ __('Author') }}</label>
+                <input name="author" type="text" class="form-control" id="author" placeholder="{{ __('Enter author') }}" value="{{ old('author') }}">
+                @if($errors->first('author'))
+                  <span class="text-danger">{{ $errors->first('author') }}</span>
+                @endif
+              </div>
+              <div class="form-group">
+                <label for="price">{{ __('Price') }}</label>
+                <input name="price" type="text" class="form-control" id="status" placeholder="{{ __('Enter status') }}" value="{{ old('price') }}">
```
Comment (at-annguyen) [inline resources/views/backend/books/create.blade.php]: What's this???

File: resources/views/backend/books/create.blade.php [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+                @if($errors->first('price'))
+                  <span class="text-danger">{{ $errors->first('price') }}</span>
+                @endif
+              </div>
+              <div class="form-group">
+                <label for="donate_by">{{ __('Donate By') }}</label>
+                <input name="donate_by" type="text" class="form-control" id="donate_by" placeholder="{{ __('Enter donater') }}" value="{{ old('donate_by') }}" >
```
Comment (at-annguyen) [inline resources/views/backend/books/create.blade.php]: donator

File: resources/views/backend/books/create.blade.php [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+              </div>
+              <div class="form-group">
+                <label for="year">{{ __('Year') }}</label>
+                <input name="year" type="text" class="form-control" id="year" placeholder="{{ __('Enter year') }}" value="{{ old('year') }}">
+                @if($errors->first('year'))
+                  <span class="text-danger">{{ $errors->first('year') }}</span>
+                @endif
+              </div>
+              <div class="form-group">
+                <label for="description">Description</label></br>
```
Comment (at-annguyen) [inline resources/views/backend/books/create.blade.php]: Localization

File: resources/views/backend/books/create.blade.php [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+                <textarea name="description" class="textarea" placeholder="{{ __('Description about this book') }}"
+                          style="width: 100%; height: 200px; font-size: 14px; line-height: 18px; border: 1px solid #dddddd; padding: 10px;"></textarea>
```
Comment (at-annguyen) [inline resources/views/backend/books/create.blade.php]: Where is id of for? 
Don't css in tag

File: resources/views/backend/books/create.blade.php [hunk 8]
```diff
@@ -0,0 +1,101 @@
...
+                <p class="help-block">{{ __('Only upload image with maximum 10mb') }}</p>
+                @if($errors->first('image'))
+                  <span class="text-danger">{{ $errors->first('image') }}</span>
+                @endif
+              </div>
+
+              <div class="box-footer">
+                <button id="btn-add-book" type="submit" class="btn btn-primary">{{ __('Submit') }}</button>
+                <button type="reset" class="btn btn-warning">{{ __('Clear') }}</button>
+              </div>
```
Comment (at-annguyen) [inline resources/views/backend/books/create.blade.php]: I think u should put back-button to come back previous url and Clear is replaced Reset

File: routes/web.php [hunk 9]
```diff
@@ -12,5 +12,10 @@
 */
 
 Route::get('/', function () {
-    return view('welcome');
+    return view('backend.layouts.main');
```
Comment (at-annguyen) [inline routes/web.php]: what... for?

**Review by annq3012 — CHANGES_REQUESTED** [review]

**Review by annq3012 — APPROVED** [review]

File: resources/views/welcome.blade.php [hunk 10]
```diff
@@ -0,0 +1,95 @@
...
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+
+        <title>Laravel</title>
+
+        <!-- Fonts -->
+        <link href="https://fonts.googleapis.com/css?family=Raleway:100,600" rel="stylesheet" type="text/css">
+
+        <!-- Styles -->
+        <style>
```
Comment (at-duyho) [inline resources/views/welcome.blade.php]: - you should move it to outer css file

**Review by ghost — CHANGES_REQUESTED** [review]
check my comment

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**